app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -302,8 +302,15 @@ val MIGRATION_9_10 = object : Migration(9, 10) {
     }
 }
 
+val MIGRATION_10_11 = object : Migration(10, 11) {
```

Comment:
What's going on here? You should probably merge from main.

Reply:
I've had the issue that after an update the app crashed and told me i need a migration to work. Tried it that way. Maybe its not correct. Sorry i am new to this 😓 
Will try to revert this.

Reply:
Haha no problem. Just merge from main and you should be good.